File: lua/test/exists_test.lua
-- Parkleitsystem SDK exists test

local sdk = require("parkleitsystem_sdk")

describe("ParkleitsystemSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)

File: ts/test/exists.test.ts

import { test, describe } from 'node:test'
import { equal } from 'node:assert'


import { ParkleitsystemSDK } from '..'


describe('exists', async () => {

  test('test-mode', () => {
    const testsdk = ParkleitsystemSDK.test()
    equal(testsdk instanceof ParkleitsystemSDK, true,
      'ParkleitsystemSDK.test() must return a client synchronously')
  })

})
